src/app.ts

```diff
@@ -0,0 +1,49 @@
+import dotenv from 'dotenv';
+import fs from 'fs';
+import http from 'http';
+import { App } from 'octokit';
+import { createNodeMiddleware, EmitterWebhookEvent } from '@octokit/webhooks';
+import {FrogbotService} from "./services/FrogbotService.js";
+import {GitHubRepo} from "./utils/types.js";
+import {webhookEvents} from "./utils/consts.js";
+
+dotenv.config();
+
+const appId: number = parseInt(process.env.APP_ID ?? '0', 0);
+const privateKeyPath: string = process.env.PRIVATE_KEY_PATH as string;
+const privateKey: string = fs.readFileSync(privateKeyPath, 'utf8');
+const secret: string = process.env.WEBHOOK_SECRET as string;
+
+const app = new App({
+  appId,
+  privateKey,
+  webhooks: {
+    secret,
+  },
+});
+
+app.webhooks.on(webhookEvents.ADD_REPOSITORIES, async ({ payload }: EmitterWebhookEvent<any>) => {
+  if (payload.action === "added") {
+    console.time('Frogbot installation time');
+    const frogbotService = new FrogbotService(await app.getInstallationOctokit(payload.installation.id));
+    const installFrogbotPromises = payload.repositories_added.map((repo : GitHubRepo) => frogbotService.installFrogbot(repo));
+    try {
+      await Promise.all(installFrogbotPromises);
```

Comment:
```suggestion
      await Promise.allSettled(installFrogbotPromises);
```
I think `all` will reject and finish if any rejected, we want to abstract one install from another right?
If think `allSettled` is the right one for this case but make sure

Reply:
i mean it changes in phase 2